src/anupSJun20/ReverseNumber.java

```diff
@@ -0,0 +1,24 @@
+package anupSJun20;
+/* Program 3: WAP to reverse number (Consider Input as Integer, not as String).
+Example:-
+Input:- 12345
+Output:- 54321*/
+
+public class ReverseNumber {
+	static void findReverseNumber(int num) {
+		int reverseNum = 0;
+		while(num > 0) {
+			reverseNum = reverseNum*10;
```

Comment:
- You can reduce the statement by following syntax:
reverseNum=reverseNum*10+(inputNum%10);

Reply:
Thanks for the comment. Incorporated the changes and pushed.